hydra/callback/callback.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Put it in hydra.core, I don't think it justifies a new top level module in Hydra.